content/blog/20240417-fiboa.md

```diff
@@ -119,37 +122,23 @@ For the initial fiboa announcement we wanted to be sure that there was at least
 
 ### Join us!
 
-So I hope this post served as a solid introduction to what Fiboa is. We'll aim to follow-up soon with more of the 'why', but if you've read all the way to here and don't think we're crazy (or think we're the right kind of crazy) then please consider joining us! We certainly can't do it all alone, and this movement is only going to succeed beyond our dreams if we manage to attract far more people than the original group. We've got a lot of great momentum, and the amazing support of Taylor Geospatial Engine, but the goal is to use these next months to really bootstrap the community that will live beyond the initial <a href="https://tgengine.org/introducing-taylor-geospatial-engine-and-the-innovation-bridge-program/&amp;sa=D&amp;source=editors&amp;ust=1713402758003405&amp;usg=AOvVaw2Lglt5UXPzh3V92tfW1_fE">Innovation Bridge initiative</a>.
-
-There's lots of places to jump in and contribute, and many were mentioned above. If you'd like to learn more than definitely join our slack or jump into the bi-weekly meetings - you're welcome to just join and observe. And if you just want to 'do something' then the best way is to actually try to take an existing field boundary dataset and try to convert it to Fiboa, and to upload it to Source Cooperative. We'll aim to have tutorials on how to do this soon, along with tags in our projects on ideas for good first tasks. But until then just jump in and ask. We look forward to working with you, and building this project together!
-
-
-----------------------------
-Don't include
```

Comment:
That should not be included. Generally, the conversion looks a bit weird with some weird HTML, footnotes, and all these tracking parts in the links?!